helm/istio/charts/sidecarInjectorWebhook/templates/meshclusterrole.yaml

```diff
@@ -9,9 +9,6 @@ metadata:
     release: {{ .Release.Name }}
     istio: sidecar-injector
 rules:
-- apiGroups: ["admissionregistration.k8s.io"]
-  resources: ["mutatingwebhookconfigurations"]
-  verbs: ["create", "get", "list", "watch", "patch"]
 # required to be able to set OwnerReference with BlockOwnerDeletion=true
```

Comment:
Same comment as above, can we remove this if we're not setting owner references?